src/DependencyInjection/GuzzleExtension.php

```diff
@@ -48,4 +48,12 @@ public function prepend(ContainerBuilder $container)
             $container->prependExtensionConfig('monolog', ['channels' => ['guzzle']]);
         }
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function getConfiguration(array $config, ContainerBuilder $container): Configuration
```

Comment:
This method wasn't marked deprecated or sth. else. 
Is there a reason for adding this method?

Reply:
Its needed since `vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Extension/Extension.php:87` can't resolve `Configuration` instance for our configuration because it has a constructor argument.